### PR TITLE
Add support to sync logicapp.json

### PR DESCRIPTION
**Pull request by suwatch**

### CODE REVIEW

File: Kudu.Core/Helpers/PostDeploymentHelper.cs [hunk 1]
```diff
@@ -75,6 +75,18 @@ private static string RoutingRunTimeVersion
...
+        // LOGICAPP_URL = [url to PUT logicapp.json to]
+        private static string LogicAppUrl
+        {
+            get { return System.Environment.GetEnvironmentVariable(Constants.LogicAppUrlKey); }
+        }
+
+        // %HOME%\site\wwwroot\logicapp.json
+        private static string LogicAppJsonFilePath
+        {
+            get { return System.Environment.ExpandEnvironmentVariables(@"%HOME%\site\wwwroot\" + Constants.LogicAppJson); }
```
Comment (davidebbo) [inline Kudu.Core/Helpers/PostDeploymentHelper.cs]: Can we rely on `Environment.FunctionsPath` here?

File: Kudu.Core/Helpers/PostDeploymentHelper.cs [hunk 2]
```diff
@@ -134,6 +146,72 @@ private static string RoutingRunTimeVersion
...
+                // for display/logging, strip out querystring secret
+                displayUrl = logicAppUrl.Substring(0, queryIndex);
+            }
+
+            var content = File.ReadAllText(fileInfo.FullName);
+            var statusCode = default(HttpStatusCode);
+            Exception exception = null;
+            try
+            {
+                var requestId = RequestIdHeader ?? Guid.NewGuid().ToString();
```
Comment (davidebbo) [inline Kudu.Core/Helpers/PostDeploymentHelper.cs]: Move NewGuid fallback logic to RequestIdHeader prop?

File: Kudu.Core/Helpers/PostDeploymentHelper.cs (hunk 1 above)
Comment (suwatch) [inline Kudu.Core/Helpers/PostDeploymentHelper.cs]: Yes we could.   However, this helper is the common routine (called by Kudu and msdeploy) so we keep it simple avoiding any more dedendency for msdeploy.

File: Kudu.Core/Helpers/PostDeploymentHelper.cs (hunk 2 above)
Comment (suwatch) [inline Kudu.Core/Helpers/PostDeploymentHelper.cs]: fixed.

**Comment by davidebbo** [comment]
Ok, I merged it in.

**Pull request merged**